src/rouge.jl

```diff
@@ -0,0 +1,152 @@
+#= ROUGE score implementation
+Lin C.Y. , 2004
+Rouge: A package for automatic evaluation of summaries
+Proceedings of the workshop on text summarization branches out (WAS 2004) (2004), pp. 25-26
+Link to paper:
+http://www.aclweb.org/anthology/W04-1013 =#
+
+using TextAnalysis
+using WordTokenizers
+function rouge_n(references, candidate, n, averaging = true)
+    #= It is a n-gram recall between a candidate summary
```

Comment:
The set of docstrings in Julia is placed above the starting of the function, You may go through this for details - https://docs.julialang.org/en/v1/manual/documentation/index.html